wandb/sdk/launch/environment/gcp_environment.py

```diff
@@ -0,0 +1,222 @@
+"""Implementation of the GCP environment for wandb launch."""
+import logging
+import os
+import re
+
+from wandb.errors import LaunchError
+from wandb.util import get_module
+
+from .abstract import AbstractEnvironment
+
+google = get_module(
+    "google",
+    required="Google Cloud Platform support requires the google package. Please"
+    " install it with `pip install google`.",
+)
+google.cloud.compute_v1 = get_module(
+    "google.cloud.compute_v1",
+    required="Google Cloud Platform support requires the google-cloud-compute package. "
+    "Please install it with `pip install google-cloud-compute`.",
+)
+google.auth.credentials = get_module(
+    "google.auth.credentials",
+    required="Google Cloud Platform support requires google-auth. "
+    "Please install it with `pip install google-auth`.",
+)
+google.auth.transport.requests = get_module(
+    "google.auth.transport.requests",
+    required="Google Cloud Platform support requires google-auth. "
+    "Please install it with `pip install google-auth`.",
+)
+google.api_core.exceptions = get_module(
+    "google.api_core.exceptions",
+    required="Google Cloud Platform support requires google-api-core. "
+    "Please install it with `pip install google-api-core`.",
+)
+google.cloud.storage = get_module(
+    "google.cloud.storage",
+    required="Google Cloud Platform support requires google-cloud-storage. "
+    "Please install it with `pip install google-cloud-storage.",
```

Comment:
IMO, we should allow users to install all of these with an extra, `wandb[launch]` and indicate it with the errors messages so thety don't hit import errors 6 times in a row

Reply:
I changed all the install messages to ask them to install wandb[launch]